api/v1alpha1/validator/nicclusterpolicy_webhook.go

```diff
@@ -659,7 +659,7 @@ func validateResources(resources map[v1.ResourceName]apiresource.Quantity, allEr
 
 // isValidOFEDVersion is a custom function to validate OFED version
 func isValidOFEDVersion(version string) bool {
-	versionPattern := `^(\d+\.\d+-\d+(\.\d+)*)$`
+	versionPattern := `^(\d+\.\d+-\d+(\.\d+)*(-\d+)?)$`
```

Comment:
nit: * -> +

we could even do it more strict e.g {3,4}

i think this release we have like 5 numbers as the patch version right @e0ne ?

so we got:

"old" version - container version not embedded: e.g 23.10-0.2.2.0
"current" version - container version embedded, dot separated: e.g 24.04-0.3.3.1.1
"future" version - container version embedded, dash separated: e.g 24.07-0.4.5.6-0

Reply:
nothing is blocker, ill let @e0ne decide.

Reply:
One thing we should definitely do is enable the webhook in the CI so issues like this are caught in future - that has to be done on the CI repo though.

Reply:
added to backlog